config/printer-creality-ender3-s1-2021.cfg

```diff
@@ -0,0 +1,128 @@
+# This file contains pin mappings for the stock 2021 Creality Ender 3
+# S1. To use this config, during "make menuconfig" select the
+# STM32F103 with a "28KiB bootloader" and serial (on USART1 PA10/PA9)
+# communication.
+
+# If you prefer a direct serial connection, in "make menuconfig"
+# select "Enable extra low-level configuration options" and select
+# serial (on USART3 PB11/PB10), which is broken out on the 10 pin IDC
+# cable used for the LCD module as follows:
+# 3: Tx, 4: Rx, 9: GND, 10: VCC
+
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
+# turning on the printer with the card inserted. The firmware
+# filename must end in ".bin" and must not match the last filename
+# that was flashed.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[stepper_x]
+step_pin: PC2
+dir_pin: PB9
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA5
+position_endstop: 0
+position_max: 220
+position_min: -5
+homing_speed: 50
+
+[stepper_y]
+step_pin: PB8
+dir_pin: PB7
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA6
+position_endstop: 0
+position_max: 220
+position_min: -5
+homing_speed: 50
+
+[stepper_z]
+step_pin: PB6
+dir_pin: !PB5
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 8
+endstop_pin: probe:z_virtual_endstop
+position_max: 270
+position_min: -3
+
+[extruder]
+step_pin: PB4
+dir_pin: PB3
+enable_pin: !PC3
+microsteps: 16
+gear_ratio: 3.5:1
+rotation_distance: 20
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PA1
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PC5
+control: pid
+pid_Kp: 22.865
+pid_Ki: 1.292
+pid_Kd: 101.178
+min_temp: 0
+max_temp: 250
+
+[heater_bed]
+heater_pin: PA7
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PC4
+control: pid
+pid_Kp: 69.139
+pid_Ki: 1.273
+pid_Kd: 938.565
+min_temp: 0
+max_temp: 130
+
+[heater_fan hotend_fan]
+pin: PC0
+heater: extruder
+heater_temp: 50.0
+
+[fan]
+pin: PA0
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB_Serial-if00-port0
+restart_method: command
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 3000
```

Comment:
3000 for untuned printers can be a bit too much, I suggest to use 1500 max for beginning.
`max_accel: 1500
max_accel_to_decel: 750
# max_accel: 3000 # after input shaping calibration you can increase to something like this or even higher
square_corner_velocity: 5`

Reply:
You are looking at an old commit, I already addressed this in the most recent version of the config, I lowered acceleration down to 2000 instead of 3000 after getting some layer shift reports.